content/tr/cloud-native-tech.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Cloud Native Teknolojisi
+status: Completed
+category: Kavram
+tags: ["Fundamental", "", ""]
+---
+
+## Nedir
+
+Cloud native yığını olarak da adlandırılan cloud native teknolojileri, cloud native uygulamaları oluşturmak için kullanılan teknolojilerdir.
+Bu teknolojiler kuruluşlar için genel, özel ve hibrit bulut ortamları gibi modern ve dinamik ortamlarda ölçeklenebilir uygulamalar oluşturmaya ve çalıştırmaya olanak tanırken [bulut bilişimin](/tr/cloud-computing/) yararlarını en üst düzeye çıkarırlar.
+Bulut bilişimin yeteneklerinden yararlanmak için sıfırdan tasarlanmışlardır ve konteynerler, servis ağları, mikro servisler ve sabit altyapı bu yaklaşımın örnekleridir.
+
+## Hangi sorunu çözer
```

Comment:
```suggestion
## Hangi Sorunları Çözer
```

We used this header in https://deploy-preview-2344--cncfglossary.netlify.app/tr/cloud-computing/.

Let's use the same for consistency.